client/src/styles/components/mj-dropdown.scss

```diff
@@ -0,0 +1,9 @@
+@use '../mistakes-journal';
+
+.dropdown-menu {
+  padding: 0;
+
+  &:focus-visible {
+    outline: 0;
+  }
+}
```

Comment:
You can also add:
```


.dropdown-item {
  &:first-child {
    @include mistakes-journal.border-top-radius(0);
  }

  &:last-child {
    @include mistakes-journal.border-bottom-radius(0);
  }
}

.dropdown-menu {
  li {
    &:first-child {
      .dropdown-item {
        @include mistakes-journal.border-top-radius(mistakes-journal.$dropdown-inner-border-radius);
      }
    }

    &:last-child {
      .dropdown-item {
        @include mistakes-journal.border-bottom-radius(mistakes-journal.$dropdown-inner-border-radius);
      }
    }
  }
}
```
It will make the border radius pretty :)